brainpy/_src/dyn/neurons/base.py

```diff
@@ -51,5 +56,26 @@ def spk_type(self):
     else:
       return self._spk_type
 
+  def offset_scaling(self, x, bias=None, scale=None):
+    s = self.scaling.offset_scaling(x, bias=bias, scale=scale)
+    if isinstance(x, bm.Variable):
```

Comment:
One more thing here need to be pay attention is:

the checking should be made on ``brainpy.math.Array``, this is because some special array instances like ``brainpy.math.SharedArray`` should keep its ``SharedArray`` property.